build.csx

```diff
@@ -202,7 +201,9 @@ public static void RunDotNetTests(IEnumerable<string> testDirectories)
 
 public static string GetVersion()
 {
-    var version = Regex.Match(File.ReadAllText(assemblyInfo), "AssemblyInformationalVersion\\(\"([^\"]+)\"\\)").Groups[1].Value;
+    var  versionInfo = FileVersionInfo.GetVersionInfo(versionAssembly);
+    var version = versionInfo.FileVersion;
+

```

Comment:
According to [this](http://stackoverflow.com/a/4466441/98713), shouldn't it be `ProductVersion` rather than `FileVersion`? Apparently `AssemblyInformationalVersion` maps to `FileVersionInfo.ProductVersion`. Not that it really matters, since we use the same value for all version attributes...